systers_portal/community/tests/selenium/test_community_admin_actions.py

```diff
@@ -0,0 +1,55 @@
+from common.tests.selenium.base import SeleniumTestCase
+from community.models import Community, RequestCommunity
+
+
+class TestCommunityAdminActions(SeleniumTestCase):
+    """
+    Extended automated visual tests for admin actions in community app,
+    test_can_create_community removed as its covered in test_admin_actions under common tests.
```

Comment:
Please remove this comment from the codebase, instead write it in the PR description. In future, if any developer sees this code, this comment would not be relevant for him/her. This comment makes sense right now, while we are freshly seeing this codebase for review and merge purposes.

Reply:
Done